### PR TITLE
Add business registration

### PR DESCRIPTION
**Pull request by Ryguy11o**
Added in a new page to help step through users in the business track registration. Needs some polishing once the user profile stuff is actually live, but take a look at what is there and let me know what you think!

### CODE REVIEW

File: website/business.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+              integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
+              crossorigin="anonymous">
+      
+        <!-- bootstrap -->
+        <link rel="stylesheet"
+              href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+              integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+              crossorigin="anonymous">
+      
+        <!-- Owl Carousel Stylesheet-->
```
Comment (sushshring) [inline website/business.html]: Remove

File: website/business.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        <!-- Compiled and minified JavaScript -->
+        <script
+            src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/js/materialize.min.js" async
+            defer></script>
+        <!--Import Google Icon Font-->
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+      
+        <!--Import Custom shared stylesheet-->
+        <link rel="stylesheet" type="text/css" href="./css/main-compiled.css">
+        <link rel="stylesheet" type="text/css" href="./css/night-main-compiled.css">
```
Comment (sushshring) [inline website/business.html]: this shouldn't be imported in here right?

File: website/business.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                            <div class="header-text text-center">
+                              <p class="text-center">Register Here</p>
+                            </div>
+                          </a>
+                        </div>
+                    </span>
+                  </li>
+                  <li>
+                    <div class="step">Step 2</div> 
+                    <span><strong>Go to your User Profile by clicking the icon in the top-right corner of the screen after registration.</strong></span></li>
```
Comment (sushshring) [inline website/business.html]:
```suggestion
                    <span><strong>Click <a href="https://app.hackpsu.org/profile>here</a> to go to sign-up for the business challenge after registration.</strong></span></li>
```
Or something like that

File: website/index.html [hunk 4]
```diff
@@ -289,7 +289,8 @@
       <p class="tracks-text">
         Choose your path at HackPSU! This year, we are introducing different challenges for those
         looking to experience HackPSU in a new way. In addition to the standard prize and challenge,
-        we are offering a design track - focused on those looking to showcase their creative flair.
+        we are offering a business track - focused on those looking to showcase their business skills
```
Comment (sushshring) [inline website/index.html]: Ooooo good catch if it wasn't changed

File: website/business.html [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+                  <li>
+                    <div class="step">Step 3</div> 
+                    <span><strong>In your User Profile, check the box that says 'Blah blah blah'</strong></span></li>
```
Comment (jmm8046) [inline website/business.html]: Maybe something more like
In your User Profile, click on the drop down menu 'Business Challenge Registration' and click the 'Register' button

File: website/business.html (hunk 2 above)
Comment (Ryguy11o) [reply website/business.html]: actually yes I am using some of their styles

Comment (Ryguy11o) [reply website/business.html]: actually yes I am using some of their styles

Comment (sushshring) [reply website/business.html]: Including night?

Comment (Ryguy11o) [reply website/business.html]: yis

**Comment by sushshring** [comment]
Remove the compiled css.map file and it looks g2g

**Pull request merged**